R/ical.R

```diff
@@ -54,20 +85,23 @@ ical <- function(x, ic_attributes = NULL) {
 ic_attributes_vec <- function(
   x = NULL,
   ic_attributes = c(
-    BEGIN = "VCALENDAR",
-    PRODID = "ATFutures/calendar",
-    VERSION = "2.0",
+    BEGIN    = "VCALENDAR",
```

Comment:
Why this extra whitespace?

Reply:
I don't know if it's redundant. But I thought it "looked better" honestly. I changed it from,

```
a = 2
aa = 2
aaa = 2
```

to 

```
a   = 2
aa  = 2
aaa = 2
```